tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=no-self-use,invalid-name
+from typing import List
```

Comment:
Some of the imports below are unused - see the pylint tests here: https://travis-ci.org/allenai/deep_qa/jobs/232628802

Just remove the ones you don't need.

You can run the pylint tests locally by running: 
`pylint -d locally-disabled -f colorized DIRECTORY`

where DIRECTORY is the directory which you want pylint to check. You should run this on deep_qa/ and tests/ as you have added code to both of these - it should help to get the formatting right.